<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagInputWidget</class>
 <widget class="QWidget" name="TagInputWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="contentsMargins">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="maximumHeight">
      <number>80</number>
     </property>
     <widget class="QWidget" name="chip_container">
      <layout class="QHBoxLayout" name="chip_layout">
       <property name="contentsMargins">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="tag_input">
         <property name="placeholderText">
          <string>새 태그 입력...</string>
         </property>
         <property name="maximumWidth">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
